commons-packet/commons-packet-service/Dockerfile

```diff
@@ -89,13 +89,15 @@ EXPOSE 8086
 CMD if [ "$is_glowroot_env" = "present" ]; then \
     wget -q --show-progress "${artifactory_url_env}"/artifactory/libs-release-local/io/mosip/testing/glowroot.zip ; \
     wget -q --show-progress "${iam_adapter_url_env}" -O "${loader_path_env}"/kernel-auth-adapter.jar; \
+    wget -q --show-progress "${cache_provider_url_env}" -O "${loader_path_env}"/cache-provider.jar; \
     wget -q --show-progress "${cache_config_url_env}" -O "${loader_path_env}"/hazelcast.xml; \
```

Comment:
when all our other code is making the cache-provider as generic, possible to change this line also as generic ?

Reply:
I thought about this, problem is - this xml is a configuration and we should be able to change it. I thought it should not be compiled in a jar.
And regarding name  - hazelcast works only when it finds hazelcast.xml in classpath. If name is anything different then it will use default hazelcast.xml which is present inside hazelcast-all dependency.